support/mailtrap.js

```diff
@@ -21,17 +21,26 @@ async function mailtrapGet(path, responseType) {
 /**
  * Get the latest Mailtrap inbox message, if any.
  *
- * @param {int} count  Number of recent emails to get.
+ * @param {string} toEmailAddress Only find emails addressed to this account
  * @returns {array}     Up to {{count}} messages, if available.
  */
-async function getLatestMessages(count = 5) {
-    const path = `/api/v1/inboxes/${process.env.MAILTRAP_INBOX_ID}/messages?search=&page=&last_id=`;
+async function getLatestMessages(toEmailAddress) {
+    if (typeof toEmailAddress !== 'string') {
+        throw new Error(`Expected toEmailAddress to be string, was ${typeof toEmailAddress}: ${toEmailAddress}`);
+    }
+
+    const params = new URLSearchParams({ search: toEmailAddress, page: '', last_id: '' });
+    const path = `/api/v1/inboxes/${process.env.MAILTRAP_INBOX_ID}/messages?${params.toString()}`;
     const messages = await mailtrapGet(path, 'json');
+
+    console.log(`got ${messages.length} message(s) from mailtrap at path ${path}`);
```

Comment:
Console log can be deleted before merge. Not committing now as I don't want to trigger another run.
```suggestion
    console.log(`got ${messages.length} message(s) from mailtrap at path ${path}`);
```